package.json

```
@@ -10,8 +10,8 @@
     "test:watch": "jest --watch",
     "start": "next start -p 8080",
     "build": "next build",
-    "lint": "next lint",
-    "lint:fix": "npm run lint -- --fix",
+    "lint": "trunk check",
+    "lint:fix": "trunk fmt",
```

```suggestion
    "lint:fix": "trunk check --fix",
    "fmt": "trunk fmt",
```
- `trunk fmt` only autoformats, which does not include eslint autofixes
- `trunk check --fix` (of `-y` for shorthand) _also_ autofixes lint errors